otter/test/test_scheduler.py

```diff
@@ -79,31 +94,32 @@ def _start_service(self):
         self.scheduler_service.running = True
 
     def test_start_service(self):
-        """
-        startService() calls super's startService() and creates SetPartitioner object
-        """
```

Comment:
Unfortunately, it also does nice things to the docstrings, like caps and punctuation, but I guess we'll just only do that for new code.